src/cli.ts

```diff
@@ -19,6 +19,18 @@ async function main(argv: string[]) {
     .option("--default-tape <tape-name>", "Name of the default tape", "default")
     .option("-h, --host <host>", "Host to proxy (not required in replay mode)")
     .option("-p, --port <port>", "Local port to serve on", "3000")
+    .option(
+      "--https-ca <filename.pem>",
+      "Enable HTTPS server with this CA certificate. Also requires --https-key and --https-cert."
+    )
```

Comment:
question: Are there any issues around invalid certificates detected by the browser (and throwing the warning screen)? A readme section would be awesome on how to use this feature.

Reply:
https://letsencrypt.org/ maybe ?

Reply:
Good point. I'll add a README around this.